task/apply-tags/0.1/apply-tags.yaml

```
@@ -0,0 +1,63 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+  name: apply-tags
+spec:
+  description: >-
+    Applies additional tags to the built image.
+  params:
+  - name: IMAGE
+    description: Reference of image that was pushed to registry in the buildah task.
+    type: string
+  - name: ADDITIONAL_TAGS
+    description: Additional tags that will be applied to the image in the registry.
+    type: array
+    default: []
+  steps:
+    - name: apply-additional-tags-from-parameter
+      image: registry.access.redhat.com/ubi9/skopeo:9.4-6@sha256:c4d70dec3eb0a0c831490192145ea25431fe04d1cf307f8d61e2d87adb41e7e3
+      args:
+        - $(params.ADDITIONAL_TAGS)
```

```suggestion
        - $(params.ADDITIONAL_TAGS[*])
```

I thought array params were supposed to be used like this https://tekton.dev/docs/pipelines/tasks/#substituting-array-parameters. Does it work the same way without the `[*]`?

---

I tested it again and the output is identical whether I use [*] or not. I noticed that yamlint task does not use it
https://github.com/mkosiarc/build-definitions/blob/main/.tekton/tasks/yaml-lint.yaml#L31

However, I'll update it so it matches the documentation